lib/Github/Client.php

```diff
@@ -77,32 +89,68 @@ class Client
      * @var array
      */
     private $options = array(
-        'base_url'    => 'https://api.github.com/',
-
-        'user_agent'  => 'php-github-api (http://github.com/KnpLabs/php-github-api)',
-        'timeout'     => 10,
-
-        'api_limit'   => 5000,
         'api_version' => 'v3',
-
-        'cache_dir'   => null
     );
 
     /**
-     * The Buzz instance used to communicate with GitHub.
+     * The object that sends HTTP messages
      *
      * @var HttpClient
      */
     private $httpClient;
 
     /**
+     * A HTTP client with all our plugins
+     *
+     * @var PluginClient
+     */
+    private $pluginClient;
+
+    /**
+     * @var MessageFactory\
+     */
+    private $messageFactory;
+
+    /**
+     * @var Plugin[]
+     */
+    private $plugins = array();
```

Comment:
indeed there's configuration for long arrays in styleci so you need to disable it